scripts/test-internal.js

```diff
@@ -86,10 +86,7 @@ function runTest(name: string, code: string): boolean {
       sourceMaps: !!sourceMap,
       additionalFunctions: additionalFunctions,
     };
-    if (name.endsWith("/bundle.js~"))
-      (options: any).additionalFunctions = [
-        "global.WildeBundle.prepareReact",
-      ];
+    if (name.endsWith("/bundle.js~")) (options: any).additionalFunctions = ["global.WildeBundle.prepareReact"];
```

Comment:
What's with the (options: any) bit?

Reply:
I'm not sure. That was already there, I just fixed the lint issue.